v22.2/backup-and-restore-overview.md

```diff
@@ -71,6 +71,16 @@ For detail on additional cloud storage features CockroachDB supports:
 
 {% include {{ page.version.version }}/misc/note-egress-perimeter-cdc-backup.md %}
 
+## Backup and restore observability
+
+You can verify that your stored backups are restorable with backup validation. A successful restore completely validates a backup, however, the validation tools offer a faster alternative and return an error message if a backup is not valid. There are three "levels" of verifying backups that give increasing validation coverage depending on the amount of runtime you want to invest in validating backups.
```

Comment:
nit: `While a successful ... a backup, the validation tool offers a faster ...`

Reply:
Yes, much smoother! I also totally forgot to add this section to v23.1 ooooops!